### ISSUE TITLE
Replaces CensoredASM?

### ISSUE THREAD

**Issue by jimmyking2004** [fonnymunkey/FermiumASM#25]
So does this replace CensoredASM as a whole and also mean this is better than CensoredASM?

**Comment by Rsslone** [comment]
No, honestly I'm not exactly sure myself as I'm still figuring out why things are the way they are, its a bit of a mess that all seems to stem from some people upset at how Rongmario branded his mods, pretending it has "pedophilic content" in it. 

They still pull all his work from loliasm anyways, so unless you're forced to use this mod through some militant arbitrary requirements, the original mod should be used.

**Comment by Nischhelm** [comment]
pretending is a fun word in this context :P

FermiumASM has various changes and improvements upon the other forks and the original. the original also got various of its fixes from other mods. FermiumASM, just as the two NormalASMs and BlahajASM, allow you to not have to use a mod in your modpack that spams "loli" everywhere. 

i get that it can feel upsetting to have a mod forked, changed and redistributed under a different name, but i hope the above reasons can make it obvious to end users why this route was chosen.

**Comment by jimmykingss** [comment]
Interesting. Well, thanks for the information; it's better getting it late then never

**Comment by Nischhelm** [comment]
woops 🙈
